webmails/start.py

```diff
@@ -43,17 +42,6 @@
     print(f"Unknown ROUNDCUBE_DB_FLAVOR: {db_flavor}", file=sys.stderr)
     exit(1)
 
-# derive roundcube secret key
-secret_key = env.get("SECRET_KEY")
-if not secret_key:
-    try:
-        secret_key = open(env.get("SECRET_KEY_FILE"), "r").read().strip()
-    except Exception as exc:
-        print(f"Can't read SECRET_KEY from file: {exc}", file=sys.stderr)
-        exit(2)
```

Comment:
As discussed in [chat](https://matrix.to/#/!MINuyJjJSrfowljYCK:tedomum.net/$1669479122105539aLcXz:matrix.org?via=tedomum.net&via=matrix.org&via=tchncs.de) I think this would break setups that use SECRET_KEY_FILE, but except the newsfragment/changelog entry it's currently undocumented anyways.

Alternatively I guess it should be also possible to move this to `base/libs/socrate/system.py`

Reply:
I've reintroduced it in 09f3cebe